src/logic/documents/scorecards.js

```diff
@@ -265,6 +266,48 @@ const scorecards = (wcif, rounds, rooms, language) => {
   return cards;
 };
 
+const shouldPrintScrambleChecker = (competitor, round, activityCode, wcif) => {
+  const { eventId } = activityCode ? parseActivityCode(activityCode) : {};
```

Comment:
You can determine it from round id, which is also an activity code (this way you don't need the extra argument):

```suggestion
  const { eventId } = activityCode ? parseActivityCode(round.id) : {};
```